tests/io/test_pyarrow_stats.py

```diff
@@ -681,6 +681,73 @@ def test_stats_types(table_schema_nested: Schema) -> None:
     ]
 
 
+def construct_test_table_without_stats() -> Tuple[pq.FileMetaData, Union[TableMetadataV1, TableMetadataV2]]:
```

Comment:
nit: this seems to be similar to `construct_test_table` above, wydt of adding a parameter to `construct_test_table` to control stat generation?
```
def construct_test_table(write_statistics: bool = True) -> Tuple[pq.FileMetaData, Union[TableMetadataV1, TableMetadataV2]]:
```

that way, we don't need this extra function

Reply:
looks like write_statistics can also take in a list 
https://arrow.apache.org/docs/python/generated/pyarrow.parquet.ParquetWriter.html

Reply:
Updated the existing function to take in a `write_statistics` argument, which can be `bool` or `list`:
```python
def construct_test_table(
    write_statistics: bool | List[str] = True,
) -> Tuple[pq.FileMetaData, Union[TableMetadataV1, TableMetadataV2]]:
```
so that it can passed directly to the `ParquetWriter`:
```python
    with pa.BufferOutputStream() as f:
        with pq.ParquetWriter(
            f, table.schema, metadata_collector=metadata_collector, write_statistics=write_statistics
        ) as writer:
            writer.write_table(table)
```